pkg/manifests/manifests.go

```diff
@@ -227,6 +244,42 @@ func newIngressControllerService(conf *config.Config) *corev1.Service {
 	}
 }
 
+func newIngressControllerServicePrivateDNS(conf *config.Config) *corev1.Service {
+	return &corev1.Service{
```

Comment:
instead of creating a new object with 90% of it being the same as `newIngressControllerService`, just modify the object that is returned with that function. It looks like you're just adding annotations.

Reply:
yep! just adding annotations. wasn't sure which would be clearer, but can definitely switch to just modifying fields in the existing object if you think that makes more sense/is more readable.